nlp_primitives/constants.py

```diff
@@ -0,0 +1,12 @@
+
+wikipedia_top_100 = pd.DataFrame({
```

Comment:
Why is `wikipedia_top_100` included here? Doesn't seem to be used anywhere else or am I missing something?

Reply:
It's not used for anything but it was mentioned in #74.  @gsheni thoughts?

Reply:
We can remove it. I don't remember specifically why I mentioned it in the issue.